JustSaying.Models/Message.cs

```diff
@@ -1,29 +1,29 @@
-using System;
+using System;
 using System.Collections.Generic;
 
-namespace JustSaying.Models
-{
-    public abstract class Message
-    {
-        protected Message()
-        {
-            TimeStamp = DateTime.UtcNow;
-            Id = Guid.NewGuid();
-        }
-
-        public Guid Id { get; set; }
-        public DateTime TimeStamp { get; set; }
-        public string RaisingComponent { get; set; }
-        public string Version{ get; private set; }
-        public string SourceIp { get; private set; }
-        public string Tenant { get; set; }
-        public string Conversation { get; set; }
-        public string ReceiptHandle { get; set; }
-        public string QueueUrl { get; set; }
+namespace JustSaying.Models
+{
+    public abstract class Message
+    {
+        protected Message()
+        {
+            TimeStamp = DateTime.UtcNow;
+            Id = Guid.NewGuid();
+        }
+
+        public Guid Id { get; set; }
+        public DateTime TimeStamp { get; set; }
+        public string RaisingComponent { get; set; }
+        public string Version{ get; private set; }
+        public string SourceIp { get; private set; }
+        public string Tenant { get; set; }
+        public string Conversation { get; set; }
+        public string ReceiptHandle { get; set; }
+        public Uri QueueUrl { get; set; }

```

Comment:
If we're changing this, we might as well rename it to `QueueUri` as well.

Reply:
Yep. let's get the build green first though.